executor/infoschema_reader.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	"context"
+	"github.com/pingcap/parser/model"
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/infoschema"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/privilege"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+	"sort"
+)
+
+// InfoschemaReaderExec executes infoschema information retrieving
+type InfoschemaReaderExec struct {
+	baseExecutor
+	iter      kv.Iterator
+	table     *model.TableInfo
+	columns   []*model.ColumnInfo
+	ChunkList *chunk.List
+	ChunkIdx  int
```

Comment:
This variables can be in lowercase, because they can be invisible to other packages.

Reply:
Done